brewtils/rest/client.py

```diff
@@ -158,13 +158,15 @@ def get_version(self, **kwargs):
         return self.session.get(self.version_url, params=kwargs)
 
     @enable_auth
-    def get_config(self, **kwargs):
+    def get_config(self, http_timeout=None, **kwargs):
```

Comment:
Since we have to support this everywhere, should we just put it in the kwargs?

What we could do is maybe search for some known kwargs to the `self.session.get` method? Just a thought for maybe later. I would at least, just search kwargs for this and put in a note in the docstring.